### ISSUE TITLE
Give warning during registration when competitor limit has been reached

### ISSUE THREAD

**Issue by EdHollingdale** [thewca/worldcubeassociation.org#3798]
This was previously raised in #1567 by Kit, although I don't think it is a direct duplicate of #328 as that has more complex questions around automated registrations. I don't feel this should be that complex after #3712.

On the "Register" tab of a competition, would it be possible to add a warning box when the competitor limit has been reached? A yellow box at the top of the page similar to the "This competition is not visible to the public" box (pre-confirmation) which says something like:

> **Note:** the competitor limit for this competition has been reached. Please read the registration requirements carefully before registering.

Whatever the message is, it should be sufficiently broad to cover different sorts of competitions that do/don't have waiting lists etc. (see #250)

**Comment by UnsolvedCypher** [comment]
It seems like that may be confusing to competitors if it's so broad. Perhaps a checkbox in the competition settings would make sense to mark whether or not the competition should have a waiting list and allow people over the limit to register?

**Comment by EdHollingdale** [comment]
I think any solution that enables people to know if the competitor limit has been reached or not is important. E.g. parents of young kids competing for the first time won't know how to check if the competitor limit has been reached. If that warning can then give them more information about waiting lists etc. then that is also a bonus.

**Comment by AlbertoPdRF** [comment]
This is probably a good first issue, right? I will label it as such!

**Comment by Jambrose777** [comment]
I'm fairly certain this is already a feature WCA provides:
![image](https://user-images.githubusercontent.com/11577241/103952423-64f37180-510e-11eb-8413-1e995dcb3a42.png)

Was implemented in #2791. Unless we want to change the text to be more broad, I think this issue can be closed.

**Comment by jonatanklosko** [comment]
Sounds about right!!
